### PR TITLE
GHSA Sync: Added 2 advisories

### PR DESCRIPTION
**Pull request by jasnow**
GHSA Sync: Added 2 advisories,

Had to change the release range on one of them to refer to the gem version, not the ruby language version.

### CODE REVIEW

File: gems/webrick/CVE-2017-10784.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+date: 2022-05-14
+description: |
+  The Basic authentication code in WEBrick library in Ruby before 2.2.8,
+  2.3.x before 2.3.5, and 2.4.x through 2.4.1 allows remote attackers to inject terminal
+  emulator escape sequences into its log and possibly execute arbitrary commands via
+  a crafted user name.
+cvss_v2: 9.3
+cvss_v3: 8.8
+patched_versions:
+  - ">= 1.4.0"
```
Comment (postmodern) [inline gems/webrick/CVE-2017-10784.yml:16]: Can you confirm that [webrick-1.4.0](https://github.com/ruby/webrick/releases/tag/v1.4.0) patches this?

File: gems/openssl/CVE-2017-14033.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Ruby OpenSSL DoS Vulnerability
+date: 2022-05-14
+description: |
+  The decode method in the OpenSSL::ASN1 module in Ruby before 2.2.8, 2.3.x
+  before 2.3.5, and 2.4.x through 2.4.1 allows attackers to cause a denial of service
+  (interpreter crash) via a crafted string.
+cvss_v2: 5.0
+cvss_v3: 7.5
+patched_versions:
+  - "> 2.0.0"
```
Comment (postmodern) [inline gems/openssl/CVE-2017-14033.yml:15]: Can you confirm that [openssl-2.0.0](https://github.com/ruby/openssl/blob/v2.0.0/History.md) patches this?

**Review by postmodern — COMMENTED** [review]
Need confirmation that the `patched_versions` are correct. The NVD information only mentions that ruby versions, not the gem versions, and I can't find mention of the vulnerability in the Release notes or ChangeLog.

File: gems/openssl/CVE-2017-14033.yml (hunk 2 above)
Comment (jasnow) [reply gems/openssl/CVE-2017-14033.yml:15]: I got the "2.0.0" data from the url: reference:
```
If you can’t upgrade Ruby itself, install OpenSSL gem newer than version 2.0.0
```

File: gems/webrick/CVE-2017-10784.yml (hunk 1 above)
Comment (jasnow) [reply gems/webrick/CVE-2017-10784.yml:16]: The commit url - https://github.com/ruby/webrick/commit/4ac0f3843ab82d1c31e1cfc719409208adef7813 (range 1.4.0 to 1.8.0) - includes a another commit - https://github.com/ruby/ruby/commit/6617c41292 - which mentions https://hackerone.com/reports/223363 which mentions the CVE.

**Comment by jasnow** [comment]
The above is all I could find.

**Review by postmodern — APPROVED** [review]
Confirmed.

**Pull request merged**